RebelAllianceBank/Classes/Bank.cs

```diff
@@ -4,11 +4,15 @@ namespace RebelAllianceBank.Classes
 {
     public class Bank
     {
-        List<IUser> users = new List<IUser>() { new Admin("FullAccessLogin", "02492512", "Admin", "Adminson") };
         IUser? currentUser;
-
+        List<IUser> users;
         public void Run()
         {
+            FileHandler fh = new FileHandler();
+            users = new List<IUser>(fh.ReadUser());
+            Customer aas = new Customer();
+            currentUser = users[0];
+            aas.TransferUserToUser(currentUser.PersonalNum, users);
```

Comment:
You seem to have left dummy data still in the code. If you intended for this then no problem, otherwise maybe you should remove that.

Reply:
Removed following code, and kept the other lines for practical use.
            Customer aas = new Customer();
            currentUser = users[0];
            aas.TransferUserToUser(currentUser.PersonalNum, users);